### ISSUE TITLE
[Feature request] Add buttons for service calls

### ISSUE THREAD

**Issue by kosmastsk** [dheera/rosboard#86]
Is there any plan for adding buttons that will provide access to ROS Service calls or even topic publishers?

What would be the necessary steps for implementing such modules for rosboard?

**Comment by dheera** [comment]
@kosmastsk Yes, I think this is very much something I would like to implement. I plan to first start by adding a teleop joystick but service calls is a good idea!

I need to think about the best way to read the service definition and generate a UI for it, but I agree it should be a very useful feature.

**Comment by jmm-slamcore** [comment]
Services and actions would be a great addition!
